qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -59,6 +59,27 @@ def measured_qubit_indices(circuit: qiskit.QuantumCircuit) -> List[int]:
     return sorted(circuit.find_bit(qubit).index for qubit in measured_qubits)
 
 
+def measured_clbit_indices(circuit: qiskit.QuantumCircuit) -> List[int]:
```

Comment:
i think this should recurse into .definitions like measured_qubit_indices does - input circuits could contain custom instructions which include measurements

Reply:
Thanks, added in [9356a04](https://github.com/Infleqtion/client-superstaq/pull/636/commits/9356a041571c7605f4d9292404ed0a5d07e7cca5) with a basic test for it in [735bb5d](https://github.com/Infleqtion/client-superstaq/pull/636/commits/735bb5dab9ada125d3b6fa5e7e702728a4d4bee0)